From 51420f8726ee97402e67d1c10c05fc33c2788a43 Mon Sep 17 00:00:00 2001
From: Miguel Bernal Marin <miguel.bernal.marin@linux.intel.com>
Date: Wed, 18 Jul 2018 04:15:32 -0500
Subject: [PATCH] media: platform: fix ADV7481 Kconfig

In the Kconfig the missing description of the int config option, set the
option as constant. By adding a description it converts to variable as
it is needed.

Signed-off-by: Miguel Bernal Marin <miguel.bernal.marin@linux.intel.com>
---
 drivers/media/platform/intel/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/media/platform/intel/Kconfig b/drivers/media/platform/intel/Kconfig
index d198d4ee69fc..383b50f17013 100755
--- a/drivers/media/platform/intel/Kconfig
+++ b/drivers/media/platform/intel/Kconfig
@@ -100,7 +100,7 @@ config INTEL_IPU4_ADV7481
 	HDMI2MIPI convertor device ADV7481
 
 config INTEL_IPU4_ADV7481_I2C_ID
-	int
+	int "I2C bus number of ADV7481 Mondello"
 	default 0
 	---help---
 	I2C bus number of ADV7481 Mondello
-- 
2.18.0

